### PR TITLE
[incubator/cassandra] Added value to configure pod tolerations

### PR DESCRIPTION
**Pull request by aleerizw-zz**
**What this PR does / why we need it**:
Adds configurable toleration fields to the cassandra spec.

### CODE REVIEW

**Comment by aleerizw** [comment]
@KongZ

File: incubator/cassandra/README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ The following table lists the configurable parameters of the Cassandra chart and
 | `service.type`             | k8s service type exposing ports, e.g. `NodePort`| `ClusterIP`                                                |
 | `podManagementPolicy`      | podManagementPolicy of the StatefulSet          | `OrderedReady`                                             |
 | `updateStrategy.type`      | UpdateStrategy of the StatefulSet               | `OnDelete`                                                 |
+| `tolerations`      | Tolerations for the pods               | `{}`                                                 |
```
Comment (KongZ) [inline incubator/cassandra/README.md]: Need to fix white-space on markdown table.

Comment (aleerizw-zz) [reply incubator/cassandra/README.md]: Done ^

**Comment by aleerizw** [comment]
@KongZ can you please review again :)

**Review by KongZ — APPROVED** [review]

**Comment by KongZ** [comment]
LGTM

**Comment by aleerizw** [comment]
@timstoop  @unguiculus Can you please review?

**Comment by aleerizw** [comment]
Ping :)

**Pull request closed without merging**